container.go

```diff
@@ -17,8 +17,9 @@ import (
 )
 
 var (
-	ErrNoImage       = errors.New("container does not have an image")
-	ErrNoRunningTask = errors.New("no running task")
+	ErrNoImage           = errors.New("container does not have an image")
+	ErrNoRunningTask     = errors.New("no running task")
+	ErrDeleteRunningTask = errors.New("cannot delete container with running task")
```

Comment:
I think it would be more accurate to change `ErrDeleteRunningTask ` to `ErrDeleteRunningContainer`, just a suggestion. : )